be/src/runtime/current_thread.h

```diff
@@ -93,6 +93,7 @@ class CurrentThread {
                 _reserved_bytes = prev_reserved;
                 _cache_size -= size;
                 _allocated_cache_size -= size;
+                _total_consumed_bytes -= size;
```

Comment:
Create one new pr to fix the problem?

Reply:
new pr: https://github.com/StarRocks/starrocks/pull/50346